google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/BigQueryAttributes.scala

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
It is possile that it is just me, but when I have multiple classes/objects in the same file, I split them to multiple files, or place the "sub"-classes into the "main" object of the file... So I would move the last 3 classes into the `BigQueryAttributes`.

Reply:
I think it's just you ;) e.g. `Some` and `None` are not inside of `Option`, same for `Either`/`Left`/`Right`.

Reply:
BTW, I just copied class structure is taken from the other connectors, `S3` IIRC.

Reply:
hmm...

https://docs.scala-lang.org/style/files.html says that `class/trait + object` should be in a file, and if you start to introduce adts you should lowercase the filename (like `option.scala`). No idea when I started to pack every subclass of a sealed trait to the companion object, but I think it started to develop when we tried out tagless final in some projects :D I told you it is nitpicking, I would pack it, you would not, and the scala codestyle says we should lowercase the filename which I would personally never do.

I can let it go :P